rtcpeerconnection.js

```diff
@@ -403,7 +406,9 @@ module.exports = function(window, edgeVersion) {
       failed: 0
     };
     this._transceivers.forEach(function(transceiver) {
-      states[transceiver.iceTransport.state]++;
+      if (!transceiver.rejected) {
```

Comment:
i ended up doing the same in #175. Mine is a bit more defensive but yours is easier in terms of branch coverage in unit tests